charts/internal/shoot-system-components/charts/csi-alicloud/templates/auth/csi-attacher-rbac.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["persistentvolumes"]
-  verbs: ["get", "list", "watch", "update"]
+  verbs: ["get", "list", "watch", "update", "patch"]
```

Comment:
It seems wired those `patch` verbs are needed when projected token feature is enabled. 
The verbs should be required according to [csi-attacher](https://github.com/kubernetes-csi/external-attacher/blob/v3.3.0/pkg/controller/csi_handler.go#L786). But it worked before without this verb.
The same behavior occured to volumeattachments. It is interesting. @rfranzke  @ialidzhikov